### PR TITLE
[zh-cn] resync: docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md [hunk 1]
```diff
@@ -87,7 +113,7 @@ CDI & MicroProfile 都会被用在互动教程中，
 
 <!-- 
 ## Example: Externalizing config using MicroProfile, ConfigMaps and Secrets
-### [Start Interactive Tutorial](/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive/) 
+[Start Interactive Tutorial](/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive/)
```
Comment (howieyuen) [inline content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md:116]: 115行后加个空行，标题与正文之间需要有空行

File: content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md [hunk 2]
```diff
@@ -87,7 +113,7 @@ CDI & MicroProfile 都会被用在互动教程中，
 
 <!-- 
 ## Example: Externalizing config using MicroProfile, ConfigMaps and Secrets
-### [Start Interactive Tutorial](/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive/) 
+[Start Interactive Tutorial](/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive/) 
 -->
 ## 示例：使用 MicroProfile、ConfigMaps、Secrets 实现外部化应用配置
-### [启动互动教程](/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive/) 
+[启动互动教程](/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive/)
```
Comment (howieyuen) [inline content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md:119]: 118行后加个空行，标题与正文之间需要有空行

File: content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md (hunk 1 above)
Comment (hs0210) [reply content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md:117]: Done.

File: content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md (hunk 2 above)
Comment (hs0210) [reply content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md:121]: Done.

**Pull request merged**